tests/test_fields_mitigate_collision.py

```diff
@@ -38,28 +38,43 @@ class TextStream(proto.Message):
     modified_obj = TestMessage({"spam": "has_spam", "eggs": "has_eggs"})
     assert modified_obj.spam_ == "has_spam"
 
-    # Test __setattr__ and __getattr___
-    modified_obj.__setattr__("spam", "no_spam")
-    modified_obj.__getattr__("spam") == "no_spam"
+    # Test get and set
+    modified_obj.spam = "no_spam"
+    assert modified_obj.spam == "no_spam"
 
-    modified_obj.__setattr__("spam_", "yes_spam")
-    modified_obj.__getattr__("spam_") == "yes_spam"
+    modified_obj.spam_ = "yes_spam"
+    assert modified_obj.spam_ == "yes_spam"
+
+    modified_obj.spam = "maybe_spam"
+    assert modified_obj.spam_ == "maybe_spam"
+
+    modified_obj.spam_ = "maybe_not_spam"
+    assert modified_obj.spam == "maybe_not_spam"
 
     # Try nested values
     modified_obj = TextStream(
         text_stream=TestMessage({"spam": "has_spam", "eggs": "has_eggs"})
     )
     assert modified_obj.text_stream.spam_ == "has_spam"
 
-    # Test __setattr__ and __getattr___
-    modified_obj.text_stream.__setattr__("spam", "no_spam")
-    modified_obj.text_stream.__getattr__("spam") == "no_spam"
+    # Test get and set for nested values
+    modified_obj.text_stream.spam = "no_spam"
+    assert modified_obj.text_stream.spam == "no_spam"
+
+    modified_obj.text_stream.spam_ = "yes_spam"
+    assert modified_obj.text_stream.spam_ == "yes_spam"
+
+    modified_obj.text_stream.spam = "maybe_spam"
+    assert modified_obj.text_stream.spam_ == "maybe_spam"
+
+    modified_obj.text_stream.spam_ = "maybe_not_spam"
+    assert modified_obj.text_stream.spam == "maybe_not_spam"
 
-    modified_obj.text_stream.__setattr__("spam_", "yes_spam")
-    modified_obj.text_stream.__getattr__("spam_") == "yes_spam"
+    with pytest.raises(AttributeError):
+        modified_obj.text_stream.attribute_does_not_exist == "n/a"
```

Comment:
Nit: should add an `assert` right at the beginning of the line.

Reply:
Done